core/Algorytm.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CTP.core
+{
+    internal class Algorytm
+    {
+        public static DataTable VtoMm(double VLow, double VHigh, double MmLow, double MmHigh, DataTable originalData)
```

Comment:
Jak będziemy chcieli konwertować kilka czujników to ta funkcja średnio sobie z tym poradzi.